docs/source/developer/contributing.rst

```diff
@@ -422,6 +439,145 @@ as to free up core developer time. Examples of ways to help:
 
 The core developers thank everyone for their understanding with the above!
 
+.. _reviewing_pull_requests:
+
+Reviewing Pull Requests
+-----------------------
+
+Help with reviewing pull requests is welcomed. There are time-effort categorised
+review tasks available on this project board: **TODO: ADD LINK**.
+
+Below are some check lists to help with reviewing pull requests that are adding,
+or altering, an implementation of some functionality e.g. adding a new NumPy
+function or updating some Python built-in. Consider selecting the check list
+appropriate to your use case and copying it into the issue as a comment, you can
+then check off the boxes as you go. Note that these check lists are a guide,
+are not exhaustive, and some parts may not always be applicable!
+
+NumPy addition/change review check list::
+
+    I have checked that:
+
+    * [ ] There is no other Pull Request proposing the same change. (If there is
+          please highlight this and one of the core developers will pick it up.)
+    * [ ] The proposed additional NumPy functionality is not deprecated by
+          NumPy.
+    * [ ] Public CI is passing.
+    * [ ] The patch does not touch files in `numba.core`.
+    * [ ] The patch is implemented using the high-level extension API only
+          (https://numba.readthedocs.io/en/stable/extending/high-level.html).
+    * [ ] The patch updates the documentation to reflect the new functionality.
+          (https://github.com/numba/numba/blob/main/docs/source/reference/numpysupported.rst)
+    * [ ] The patch is line wrapped to 80 characters.
```

Comment:
Need to add a note about running a "spell checker", and making sure that NumPy/SciPy/Python are correctly capitalised.

Reply:
hm... can we automate these type of checks as part of CI?